cadc-vos/src/main/java/org/opencadc/vospace/Node.java

```diff
@@ -211,4 +212,33 @@ public String getPropertyValue(URI uri) {
         return null;
     }
 
+    /**
```

Comment:
Don't really want convenience methods in the "data model" classes unless the logic has to be internal... these are either wrong or return different results depending on context: please remove from Node.